src/apiDefs/env.ts

```diff
@@ -18,3 +18,12 @@ export const getEnvFlags = () => {
   }, {});
   return envFlags;
 };
+
+export const getCategoryFlags = () => {
+  const apiDefinitions = getApiDefinitions();
+  const categories = {};
+  getApiCategoryOrder().forEach((category) => {
+    categories[category] = apiDefinitions[category].apis.filter( api => isHostedApiEnabled(api.urlFragment, api.enabledByDefault) ).length > 0;
```

Comment:
```suggestion
    categories[category] = apiDefinitions[category].apis.some( api => isHostedApiEnabled(api.urlFragment, api.enabledByDefault) );
```

slightly shorter syntax if you'd like to use it. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some